zbus/src/connection/mod.rs

```diff
@@ -748,7 +750,7 @@ impl Connection {
                             }
                         }
                     }
-                    .instrument(info_span!("{}", task_name)),
+                    .instrument(info_span!("monitor_name_acquired", name = %well_known_name_clone)),
```

Comment:
You can't avoid the cloning by creating the span earlier instead (where `task_name` is being created)? :thinking: You could name it `task_name_span` (for example). Same for the other clone above.